api/tests/unit/application/schooling-registration-user-associations/index_test.js

```diff
@@ -16,52 +15,29 @@ describe('Unit | Application | Router | schooling-registration-user-associations
   const studentId = '1234';
   const userId = 2;
 
-  let httpTestServer;
-
-  beforeEach(async() => {
-    sinon.stub(preHandler, 'checkUserIsAdminInSUPOrganizationManagingStudents');
-
-    sinon.stub(schoolingRegistrationUserAssociationController, 'dissociate').returns('ok');
-    sinon.stub(schoolingRegistrationUserAssociationController, 'updateStudentNumber').returns('ok');
-
-    httpTestServer = new HttpTestServer();
-    await httpTestServer.register(moduleUnderTest);
-  });
-
   describe('PATCH /api/organizations/id/schooling-registration-user-associations/studentId', () => {
 
     const method = 'PATCH';
     const headers = {
       authorization: generateValidRequestAuthorizationHeader(userId),
     };
 
-    let payload;
-    let url;
-
-    beforeEach(() => {
-      payload = {
-        data: {
-          attributes: {
-            'student-number': '1234',
-          },
-        },
-      };
-    });
-
-    afterEach(() => {
-      preHandler.checkUserIsAdminInSUPOrganizationManagingStudents.restore();
-    });
-
     context('when the user is authenticated', () => {
 
-      beforeEach(() => {
-        preHandler.checkUserIsAdminInSUPOrganizationManagingStudents.callsFake((request, h) => h.response(true));
-      });
-
       it('should exist', async () => {
         // given
-        url = `/api/organizations/${organizationId}/schooling-registration-user-associations/${studentId}`;
-
+        sinon.stub(preHandler, 'checkUserIsAdminInSUPOrganizationManagingStudents').callsFake((request, h) => h.response(true));
+        sinon.stub(schoolingRegistrationUserAssociationController, 'updateStudentNumber').returns('ok'); const httpTestServer = new HttpTestServer();
```

Comment:
```suggestion
        sinon.stub(schoolingRegistrationUserAssociationController, 'updateStudentNumber').returns('ok');
        const httpTestServer = new HttpTestServer();
```

**Nitpick : ** Ajout d'un saut de ligne avant la déclaration du serveur.